clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -190,6 +190,32 @@ const inline float dot2add(half2 A, half2 B, float C) {
   return __detail::dot2add_impl(A, B, C);
 }
 
+//===----------------------------------------------------------------------===//
+// dst builtins
+//===----------------------------------------------------------------------===//
+
+/// \fn fvector dst(fvector, fvector)
+/// \brief Calculates a distance vector.
+/// \param Src0 [in] The first vector contains the squared distance represented
+/// as {_, d*d, d*d, _}
```

Comment:
This needs to be shorter also what is d in this case. Introducing a variable and not defining it is going to be confusing to users.  
```suggestion
/// \param Src0 [in] Contains the squared distance
```

Reply:
Sure, I will shorten the description. However, d represents the distance, and the argument contains the square of the distance.